usr/src/uts/common/fs/zfs/zil.c

```diff
@@ -1057,12 +1061,13 @@ zil_lwb_write_start(zilog_t *zilog, lwb_t *lwb)
 static lwb_t *
 zil_lwb_commit(zilog_t *zilog, itx_t *itx, lwb_t *lwb)
 {
-	lr_t *lrc = &itx->itx_lr; /* common log record */
-	lr_write_t *lrw = (lr_write_t *)lrc;
+	lr_t *lrcb, *lrc = &itx->itx_lr; /* common log record */
```

Comment:
Why do we need the two new variables (`lrcb` and `lrwb`)?

I see them used below, but it's not obvious why we needed these new variables, opposed to using the existing variables like the previous code did; I'm not sure it matters one way or another, but wanted to ask to clear up my understanding.